warp10/src/main/java/io/warp10/script/processing/image/Pdecode.java

```diff
@@ -79,21 +63,10 @@ public Object apply(WarpScriptStack stack) throws WarpScriptException {
     
     PImage image = new PImage(awtImage);
     
-    //
-    // Check transparency
-    //
+    //The transparency test in Pimage(awtImage) is not working correctly: set format to ARGB is mandatory to keep transparency.
```

Comment:
I am not sold on this comment, the question is whether or not the behavior is different when forcing the image format to ARGB